scripts/build_local_image.sh

```diff
@@ -19,8 +19,13 @@ source "$RELAYER_PATH"/scripts/versions.sh
 full_commit_hash="$(git --git-dir="$RELAYER_PATH/.git" rev-parse HEAD)"
 commit_hash="${full_commit_hash::8}"
 
+./scripts/build.sh
```

Comment:
Was this broken previously? Seems unrelated to the rest of this PR

Reply:
Sorry meant to add a comment here explaining this. We removed the build stage from the Dockerfile [here](https://github.com/ava-labs/awm-relayer/pull/188/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557), in favor of goreleaser's approach which builds the binary outside of the image. This change should have been included in that PR.